lib/jekyll-feed.rb

```diff
@@ -12,6 +12,14 @@ def config
       @context.registers[:site].config
     end
 
+    def path
+      if config["feed"] && config["feed"]["path"]
```

Comment:
Will this blow up if someone does `feed: foo` in their config file? In that case `config["feed"]` won't be a dict.